src/mapgen/cavegen.cpp

```diff
@@ -69,7 +69,7 @@ CavesNoiseIntersection::~CavesNoiseIntersection()
 
 
 void CavesNoiseIntersection::generateCaves(MMVManip *vm,
-	v3s16 nmin, v3s16 nmax, u8 *biomemap)
+	v3s16 nmin, v3s16 nmax, u16 *biomemap)
```

Comment:
this should rather use `biome_t` to prevent any future mistakes

Reply:
Will do this as a last step once everything works.

Reply:
Done everywhere possible.